rs/registry/admin/src/main.rs

```diff
@@ -5562,7 +5562,7 @@ async fn propose_to_install_code(
         Some(proposer),
     ));
 
-    let canister_id = Some(PrincipalId::try_from(cmd.canister_id).unwrap());
+    let canister_id = Some(cmd.canister_id.get());
```

Comment:
NCR

I like PrincipalId::from(cmd.canister_id) better. More explicit. I would not be able to confidently guess what CanisterId::get does.

OTOH, `let canister_id = ` is already telling us that this is just some kind of conversion that's needed only to make types match, and so the details of what the new type is do not matter very much.

Reply:
> is already telling us that this is just some kind of conversion that's needed
Yes that's my reasoning as well. Otherwise `::from` sounds good